FirebaseAuth/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
 # Unreleased
 - [changed] Added a `X-Firebase-GMPID` header to network requests.
+- [fixed] macOS Extension access to Shared Keychain by adding kSecUseDataProtectionKeychain recommended key (#8950, #6876)
```

Comment:
Hi, could you remove the #8950 part from the changelog entry, please? Reason being is that this patch fixes Auth's keychain access but I believe we still need to investigate the root cause for the `com.firebase.FIRInstallations.installations`-related prompts, which may be a separate fix.

```suggestion
- [fixed] macOS Extension access to Shared Keychain by adding kSecUseDataProtectionKeychain recommended key (#6876)
```

Reply:
Done, thank you.